examples/stream-producer.js

```diff
@@ -6,7 +6,6 @@ async function streamProducer() {
   const client = createClient();
 
   await client.connect();
-  await client.del('mystream');
```

Comment:
?

Reply:
@leibale I removed the `del` because this makes this example work better with the consumer group one... if this deletes the stream it will delete any associated consumer groups too and then if they're running the consumer group example and it runs out of entries to process then they start the producer again it will blow away the stream rather than just continue to add new entries to it.